decoder.py

```diff
@@ -90,6 +94,26 @@ def decode_logical_screen_descriptor(gif_stream: BitStreamReader, gif_object: Gi
     pixel_ratio_value = gif_stream.read_unsigned_integer(1, 'bytes')
     gif_object.pixel_aspect_ratio = (pixel_ratio_value + 15) / 64
 
+def background_frame(gif_object: Gif):
+    prev_image = gif_object.images[-1]
+    current_image = Image()
+    current_image.index_graphic_control_ex = None
+
+    current_image.left = prev_image.left
+    current_image.top = prev_image.top
+    current_image.width = prev_image.width
+    current_image.height = prev_image.height
```

Comment:
r u sure we not need the full size of the gif?

Reply:
[from what i have seen](http://www.theimage.com/animation/pages/disposal2.html), only the frame part is need to be returned to background